src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -73,7 +73,7 @@
   <ItemGroup>
```

Comment:
Please revert this unchanged file `git checkout master src/DynamoCoreWpf/DynamoCoreWpf.csproj`